### PR TITLE
Aegis gear updoot

### PR DESCRIPTION
**Pull request by RadiantFlash7**
## About The Pull Request
Cool new stuff:
New Aegis armor type called CAS Assault Harness, reduced armor for storage, placed in aegis prep
New aegis turtleneck, added to fashion closets
Cobalt Breaching hammer, used to smash down normal walls. Currently held and issuied by the sergeant.

## Why It's Good For The Game
Cause... fashion souls ss13 edition. And good features?

## Changelog
```changelog
add: New Aegis armor type called CAS Assault Harness
add: Aegis turtleneck
Add: CAS Breaching hammer
add: Adds objs to map!
```

### CODE REVIEW

File: zzzz_modular_occulus/code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -25,3 +25,31 @@
...
+		/obj/item/device/lighting/toggleable/flashlight,
+		/obj/item/modular_computer/pda,
+		/obj/item/device/radio/headset,
+		/obj/item/device/hailer,
+		/obj/item/device/megaphone,
+		/obj/item/weapon/melee,
+		/obj/item/weapon/gun/projectile/selfload,
+		/obj/item/weapon/gun/projectile/giskard,
+		/obj/item/weapon/gun/energy/gun/martin,
+		/obj/item/taperoll,
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/objects/items/weapons/storage/belt.dm:54]:
```suggestion
		/obj/item/taperoll
```

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]
CI's failing

**Comment by Shadow-Quill** [comment]
Also, `mapchange` is not a valid changelog tag

File: zzzz_modular_occulus/code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/game/objects/items/weapons/storage/belt.dm:54]: oops.

Comment (RadiantFlash7) [reply zzzz_modular_occulus/code/game/objects/items/weapons/storage/belt.dm]: Fixed!

**Comment by RadiantFlash7** [comment]
I don't know why CI is failing

File: cev_eris.dme [hunk 2]
```diff
@@ -2910,6 +2910,7 @@
 #include "zzzz_modular_occulus\code\game\objects\items\weapons\storage\firstaid.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\weapons\storage\pouches.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\weapons\storage\storage.dm"
+#include "zzzz_modular_occulus\code\game\objects\items\weapons\tools\cobalthammer.dm"
```
Comment (Shadow-Quill) [inline cev_eris.dme:2913]: CI is failing because of this addition, but this file isn't included in the PR.

**Comment by Shadow-Quill** [comment]
You've got `/obj/item/weapon/tool/hammer/CAS` mapped in

**Review by Shadow-Quill — APPROVED** [review]

**Pull request merged**